net/shadowsocks/src/opnsense/mvc/app/models/OPNsense/Shadowsocks/General.xml

```diff
@@ -46,5 +46,14 @@
                 <chacha20-ietf-poly1305>ChaCha20-IETF-Poly1305</chacha20-ietf-poly1305>
             </OptionValues>
         </cipher>
+        <tcpudpmode type="OptionField">
+            <default>tcp_only</default>
+            <Required>Y</Required>
```

Comment:
If you introduce a new "Required" field, you have to increment the model version too (e.g. 1.0.1). Otherwise existing configurations won't be migrated to include this new field.